common/configuration/1password_provider.go

```diff
@@ -28,21 +29,25 @@ func (o OnePasswordProvider) Delete(ctx context.Context, ref Ref) error {
 	return nil
 }
 
+func (o OnePasswordProvider) itemName() string {
+	return o.ProjectName + ".secrets"
```

Comment:
Does this need the `"ftl." +` prefix, or does that get added somewhere else?

Reply:
I moved away from the `ftl.` prefix, but i hadn't updated the documentation. good catch!